src/index.html

```diff
@@ -22,6 +22,333 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#moyoLogo"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Company Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <li>
+          <a
+            class="nav__link is-active"
+            href="#apple"
+          >
+            apple
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#samsung"
+          >
+            samsung
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#smartphones"
+          >
+            smartphones
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            data-qa="nav-hover"
+            href="#laptops_and_computers"
+          >
+            laptops & computers
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#gadgets"
+          >
+            gadgets
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#tablets"
+          >
+            tablets
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#photo"
+          >
+            photo
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#video"
+          >
+            video
+          </a>
+        </li>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__imac-image"
+          src="./images/imac.jpeg"
+          alt="imac-image"
+        />
+        <div class="card__content">
+          <h3 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+            <div class="card__price-tag">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
```

Comment:
It seems like you are repeating the same card structure multiple times. It would be a good idea to use a loop to generate these cards if you are using a templating engine or JavaScript. This will make your code more maintainable and less prone to errors.